CHANGELOG.rst

```diff
@@ -1,20 +1,28 @@
 Changelog
 =========
 
-* 0.8.1
+* 0.9.1
+    * dhcp: a huge upgrade, all the protocol module reimplemented using asyncio
+    * 9p: a basic client / server / fs implementation (no 9p2000.L yet)
+    * fixtures: added public pytest fixtures
+    * core: rewritten using asyncio
```

Comment:
I think that it should be the first line, since it has probably the biggest impact (and possible side effects)